entprom/go.mod

```diff
@@ -0,0 +1,10 @@
+module entprom
```

Comment:
remove this file, use the global one with everyone else. we will split the modules if needed at a later point in time